frontend/lib/modules/lend_library/lend_library.dart

```diff
@@ -168,12 +173,28 @@ class _LendLibraryState extends State<LendLibrary> {
 
   void _init() async {
     if (!_skipCurrentLocation) {
-      var coordinates = await _location.getLocation();
-      if (coordinates.latitude != null) {
+      var coordinates;
+      UserClass? _currentUser = Provider.of<CurrentUserState>(context, listen: false).currentUser;
```

Comment:
Do you think this bit of code may be more generically useful (e.g. in CurrentUserState or a new location service?). Not required, but an idea.

Reply:
Good point, I'm moving it into a function in CurrentUserState. I can easily see it being handy somewhere else down the road